src/util/hash.rs

```diff
@@ -123,6 +123,12 @@ impl_array_newtype!(Sha256Sum, u8, 32);
 impl_array_hexstring_fmt!(Sha256Sum);
 impl_byte_array_newtype!(Sha256Sum, u8, 32);
 
+impl Default for Sha256Sum {
```

Comment:
Is there a reason we need this?

Reply:
`struct RPCHandlerArgs<'a>` does not compile without it.